code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm

```diff
@@ -0,0 +1,370 @@
+/mob/living/simple_animal/hostile/megafauna/colossus
+	name = "colossus"
+	desc = "A monstrous creature protected by heavy shielding."
+	health = 2500
+	maxHealth = 2500
+	attacktext = "judges"
+	attack_sound = 'sound/magic/clockwork/ratvar_attack.ogg'
+	icon_state = "eva"
+	icon_living = "eva"
+	icon_dead = "dragon_dead"
```

Comment:
I don't have the rest of the sprites yet also it deletes on death

I don't think 96x96 is small and unimposing either.